apps/demo-typegen/src/demo.test.ts

```diff
@@ -10,19 +10,21 @@ import type { PredicateAbiInputs } from './predicate-types';
 import { PredicateAbi__factory } from './predicate-types';
 import { ScriptAbi__factory } from './script-types';
 
-let baseAssetId: string;
-
 /**
  * @group node
+ * @group browser
  */
 describe('ExampleContract', () => {
-  beforeAll(async () => {
-    const provider = await Provider.create(FUEL_NETWORK_URL);
-    baseAssetId = provider.getBaseAssetId();
-  });
   it('with imported storage slots', async () => {
-    const provider = await Provider.create(FUEL_NETWORK_URL);
-    const wallet = await generateTestWallet(provider, [[500_000, baseAssetId]]);
+    using launched = await launchTestNode({
+      walletsConfig: {
+        amountPerCoin: 500_000,
+      },
+    });
```

Comment:
```suggestion
    using launched = await launchTestNode();
```

[Default](https://github.com/FuelLabs/fuels-ts/blob/7ae80b8a4af7a2b770bb687edf1d9db671cc4a60/packages/account/src/test-utils/setup-test-provider-and-wallets.ts#L33) `amountPerCoin` is `10_000_000_000` so you don't need to define this.

Reply:
755f8e0